readme.md

```diff
@@ -59,6 +59,19 @@ Default: `true`
 
 Overwrite existing destination file.
 
+##### cwd
+
+Type: `string`\
+Default: `process.cwd()`
+
+When your source, destination path are relative, `cp-file` basically finds out the source, destination paths from the path you executed the process (`process.dir()`)
+
+You can change this base path to which path you want.
+
+This ensures `cp-file`'s behavior be same regardless of your path executed the process.
+
+When your source, destination path are not relative, this option is ignored.
+
```

Comment:
The docs here are still not good enough. I would recommend starting over. Try to succinctly explain what it does. You shouldn't need more than a couple of short sentences.